### DISCUSSION TITLE
node-red-contrib-noble-bluetooth lacks module bluetooth-hci-socket

### DISCUSSION THREAD

**Discussion by 8xiom** [ct-Open-Source/ct-Smart-Home#45] [Q&A] [answered]
First of all a big 'thank you' to Merlin for releasing the new version shorty before Christmas. The update process worked without problems, except for one issue.
Unfortunately the long awaited solution for bluetooth support doesn't work with my setup on a Raspi 3B+ running Debian Buster. The error message (in fact it's labeled '[warn]') points towards a specific module:

_cannot find module '@abandonware/bluetooth-hci-socket', Require stack:, [..]_

Did anyone else encounter this problem?

Best regards,
Bernd

**Comment by merlinschumacher** [comment, marked as answer]
Hi, 

it seems the `bluetooth-hci-socket` module is missing. I've created a pull request (#46) that fixes the issue. The next update will have working Bluetooth support again.

best regards

Merlin

**Reply by 8xiom** [reply]
Works fine now, thanks!
